lib/command-runner.js

```diff
@@ -1,20 +1,34 @@
 'use strict';
 const {spawn} = require('child_process');
+const {join} = require('path');
 const which = require('which');
+const fs = require('fs');
+const path = require('fs');
 
 exports.run = function (command) {
   return new Promise((resolve, reject) => {
     which(command.cmd, (err, cmdpath) => {
```

Comment:
A better solution would be to use which's `path` option for this.
So something like this:

```js
const {join, delimiter} = require('path');
...
which(command.cmd, {path: `${join('node_modules', '.bin')${delimiter}${process.env.PATH}`}, ...
```